### ISSUE TITLE
googlefonts/axisregistry release schedule

### ISSUE THREAD

**Issue by chrissimpkins** [googlefonts/axisregistry#85]
Who uses the GitHub releases?  And how frequently should they take place?

**Comment by felipesanches** [comment]
GitHub releases result in a version tag being created on the git repo. The tag then automatically triggers a continuous integration job that creates the python package and published it on PyPI. And these packages are used by all other tools (such as gftools and fontbakery).

I would advocate for such releases to be made frequently, whenever some relevant change is made to the axisregistry repo. One good thing about this release scheme is that it allows the tools to be kept in sync. The same axis-registry data is used on all tools (and enforced by fontbakery checks) as long as users install the latest versions of the tools.

**Comment by chrissimpkins** [comment]
Ty! Who has maintained releases to date?

**Comment by RosaWagner** [comment]
I made the last one, a new one is needed?

**Comment by felipesanches** [comment]
yes, a new axisregistry release is needed asap.

Here's the reasoning:
* Axisregistry has a too strong requirement on setup.py for protobuf == 3.19.4
* It clashed with my attempt of using a newer version of protofub on fontbakery.
* I downgraded protobuf version on fontbakery in order to release FB 0.8.11a2
* I made a commit on axisregistry repo to remove the strong requirement

Also, here's some good practices that I learned in the past from @anthrotype (and have been using on FontBakery and other python projects):
* Dependencies on setup.py should be kept relaxed to facilitate usage of modules on other projects
* Dependencies on requirements.txt should be specific, to document the exact set of dep-versions used during development and code-testing.
![Screenshot from 2022-10-26 17-21-54](https://user-images.githubusercontent.com/213676/198299752-92592292-08ea-4b47-b6d4-3920474adca6.png)
